src/wazuh_db/wdb_integrity.c

```diff
@@ -229,7 +230,7 @@ void wdbi_update_attempt(wdb_t * wdb, wdb_component_t component, long timestamp)
  * @param timestamp Synchronization event timestamp (field "id");
  */
 
-static void wdbi_update_completion(wdb_t * wdb, wdb_component_t component, long timestamp) {
+void wdbi_update_completion(wdb_t * wdb, wdb_component_t component, long timestamp, os_sha1 manager_checksum) {
```

Comment:
This new parameter is not mentioned in the method's doxygen

Reply:
It was added in the Doxygen of wdb.h.
As there is a doxygen on wdb.h, I will remove this duplicate in wdb_integrity.c